MaterialLib/FractureModels/Permeability/CreateConstantPermeability.cpp

```diff
@@ -0,0 +1,38 @@
+/**
+ * \file
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "CreateConstantPermeability.h"
+#include "BaseLib/ConfigTree.h"
+#include "BaseLib/Error.h"
+
+#include "ConstantPermeability.h"
+
+namespace MaterialLib::Fracture::Permeability
+{
+std::unique_ptr<Permeability> createConstantPermeability(
+    BaseLib::ConfigTree const& config)
+{
+    //! \ogs_file_param{prj__processes__process__HYDRO_MECHANICS_WITH_LIE__fracture_properties__permeability_model__type}
```

Comment:
As my comment to #2542, it is better to change the path of doc as
`//! \ogs_file_param_special{material__porous_medium__fracture_properties__permeability_model__type}`.